frontend/src/app/services/mitigations.service.ts

```diff
@@ -188,8 +188,10 @@ export class MitigationsService {
       .subscribe(diff => this.set(diff.newValue));
 
     this.setValue$
-      .pipe(untilDestroyed(this))
-      .subscribe(preset => this.set(preset[1]));
+      .pipe(
+        startWith(['reset', gameService.game.mitigations] as PresetValue),
```

Comment:
@JaroslavSoldat @Petr26 do you think this might cause any problem?

Reply:
It is ok IMO.
Isn't it duplicated by gameService.reset$ below?
Never mind, rather twice than not set.